src/main/app/src/app/formulieren/model/goedkeuren.ts

```diff
@@ -107,8 +107,9 @@ export class Goedkeuren extends AbstractFormulier {
 
     getGoedkeurenOpties(): Observable<string[]> {
         return of([
-            'actie.ja',
-            'actie.nee'
+            'actie.goedkeuren.akkoord',
```

Comment:
Gebruik een enum

Reply:
We doen het ook op deze manier in `AanvullendeInformatie` en in zekere zin in `IntakeAfrondenDialogComponent`. Moeten deze usages dan ook herschreven worden met een enum? Zou zeker geen kwaad kunnen, maar ik weet niet of het veel meerwaarde heeft (helemaal voor een issue met een weight van 1 )

Reply:
Enum gebuikt in nieuwe commit.